packages/support/resources/views/components/section/index.blade.php

```diff
@@ -163,10 +164,10 @@
         <div
             @class([
                 'fi-section-content',
-                'rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10' => $aside,
+                'rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10' => $aside && (! $minimal),
                 match ($compact) {
                     true => 'p-4',
-                    false => 'p-6',
+                    false => ! $contained ? 'pt-4' : 'p-6',
```

Comment:
Also get rid of the match expression here, because this gets quite unreadable otherwise. 🙂